Sming/Makefile-windows.mk

```diff
@@ -9,7 +9,7 @@ SDK_BASE	?= $(ESP_HOME)/ESP8266_SDK
 SDK_TOOLS	 ?= $(ESP_HOME)/utils
 
 # Other tools mappings
-ESPTOOL		 ?= $(SDK_TOOLS)/esptool.exe
+ESPTOOL		 ?= $(SDK_TOOLS)/ESP8266/esptool.exe
```

Comment:
I have to double check but as far as I can see SDK_TOOLS is used only for the tools directory, which means that in you case where it is under SDK_TOOL/ESP8266 you can define it directly in your environment without the need to change it in the Sming source code (as @anakod correctly noticed ( #938) and complained about it ). 

@ADiea Do you know if there is a way to get the UDK version? We could add conditions that set correctly the SDK_TOOLs directory for UDK on Windows.